src/main/java/com/google/opensesame/servlets/MentorsServlet.java

```diff
@@ -26,11 +26,14 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws ServletException, IOException {
+    addMockMentor(response);
     List<UserEntity> mentorEntities = ofy().load().type(UserEntity.class).list();
     ArrayList<UserData> mentors = new ArrayList<UserData>();
     for (UserEntity entity : mentorEntities) {
       if (entity.isMentor()) {
-        mentors.add(new UserData(entity));
+        UserData mentorData = new UserData(entity);
+        mentorData.getProjects();
```

Comment:
It seems like this is just a getter function. Is there a reason that this is being called here?

Reply:
good catch, that was a call to deprecated functionality